stacks/core/prod/traefik.yml

```diff
@@ -0,0 +1,68 @@
+version: '3.7'
+
+services:
+  users:
+    deploy:
+      labels:
+        - traefik.enable=true
+        - traefik.http.services.users.loadbalancer.server.port=8180
+        - traefik.http.routers.users.rule=HostRegexp(`{subdomain:users}.{domain:[a-zA-Z0-9.]+}`)
+        - traefik.http.routers.users.entrypoints=http
+
+  things:
+    deploy:
+      labels:
+        - traefik.enable=true
+        - traefik.http.services.things.loadbalancer.server.port=8182
+        - traefik.http.routers.things.rule=HostRegexp(`{subdomain:things}.{domain:[a-zA-Z0-9.]+}`)
+        - traefik.http.routers.things.entrypoints=http
+
+  authn:
+   deploy:
+      labels:
+        - traefik.enable=true
+        - traefik.http.services.authn.loadbalancer.server.port=8183
+        - traefik.http.routers.authn.rule=HostRegexp(`{subdomain:authn}.{domain:[a-zA-Z0-9.]+}`)
+        - traefik.http.routers.authn.entrypoints=http
+
+  storage:
+    deploy:
+      labels:
+        - traefik.enable=true
+        - traefik.http.services.storage.loadbalancer.server.port=8181
+        - traefik.http.routers.storage.rule=HostRegexp(`{subdomain:storage}.{domain:[a-zA-Z0-9.]+}`)
+        - traefik.http.routers.storage.entrypoints=http
+
+  babeltower:
+    deploy:
+      labels:
+        - traefik.enable=true
+        - traefik.http.services.babeltower.loadbalancer.server.port=8080
+        - traefik.http.routers.babeltower.rule=HostRegexp(`{subdomain:api}.{domain:[a-zA-Z0-9.]+}`)
+        - traefik.http.routers.babeltower.entrypoints=http
+
+  rabbitmq:
```

Comment:
The ideal is running it with the management plugin disabled and do all configurations via CLI.
TODO: investigate which ports must be opened.